.github/workflows/pr-change-set.yaml

```diff
@@ -0,0 +1,21 @@
+# This workflow checks and tests the package code, and it builds all package
+# artifacts whenever there were changes to a pull request.
+
+name: 'Pull Request: change set'
```

Comment:
I think `Pull Request` in the name is not necessary because the trigger should appear in the workflow run.

How about "Check change set"?

Reply:
On the website the action run shows the `name:` of the Action:

![actions](https://user-images.githubusercontent.com/12053937/195204352-6fda347c-c950-4996-b02d-801fd1d68926.jpg)

with

https://github.com/jenstroeger/python-package-template/blob/e76f2cdc40a566205c6dfedc07b004f5617261db/.github/workflows/scorecards-analysis.yaml#L3

> How about "Check change set"?

I worry that the run log shows only that, and we don’t have the context that the check runs on a PR?

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/336/commits/bd8d88b09ac1f71ab8d4c78c0a8c7335e3536f94.